### PR TITLE
More tweaks to the readme.

### PR DESCRIPTION
**Pull request by tomwilkie**
Just running through this.

Also want to show example of how to override the selectors, but that can wait.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,20 +9,30 @@ A set of Grafana dashboards and Prometheus alerts for Kubernetes.
...
-commands:
+You can manually generate the alerts, dashboards and rules files, but first you
+must install some tools:
+
+```
+$ go get github.com/jsonnet-bundler/jsonnet-bundler/cmd/jb
+$ brew install jsonnet
+```
+
+Then, grab the mixin and its dependancies:
```
Comment (brancz) [inline README.md]: dependencies

**Comment by tomwilkie** [comment]
@brancz added docs on customisation too, PTAL

File: README.md [hunk 2]
```diff
@@ -97,6 +108,39 @@ $ ks apply default
...
+
+Kubernetes-mixin allows you to override the selectors used for various jobs,
+to match those used in your Prometheus set.
+
+In a new directory, add a file `mixin.libsonnet`:
+
+```
+local kubernetes = import "kubernetes-mixin/mixin.libsonnet";
+
+kubernetes {
```
Comment (brancz) [inline README.md]: should this be `kubernetes + {` ?

File: README.md [hunk 3]
```diff
@@ -97,6 +108,39 @@ $ ks apply default
...
+  _config+:: {
+    kube_state_metrics_selector: 'job="kube-state-metrics"',
+    cadvisor_selector: 'job="kubernetes-cadvisor"',
+    node_exporter_selector: 'job="kubernetes-node-exporter"',
+    kubelet_selector: 'job="kubernetes-kubelete"',
```
Comment (brancz) [inline README.md]: `kubelete` -> `kubelet`

File: README.md (hunk 2 above)
Comment (tomwilkie) [reply README.md]: what i've put in is shorthand - they do the same.

**Pull request merged**